src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,190 @@
 layout: content.html
 ---
 
+<%
+// Plugin data
+// TODO: put badges into badge object to iterate over
+// Badges are all seneca-gateway temporarily
+let pluginsByGroup = [
+  {
+    gateway: [
+      {
+        name: 'gateway-lambda',
+        title: 'gateway-lambda',
+        desc: 'Handle incoming messages within other frameworks.',
+        github_link: 'https://github.com/senecajs/seneca-gateway-lambda/',
```

Comment:
this data will still have to entered and maintained by hand, so there's a lot of scope to avoid repetition by automatically prepending known websites etc.

example:
```
github_link: 'senecajs/seneca-gateway-lambda'
```